Gemfile

```diff
@@ -71,7 +71,7 @@ group :test do
   gem 'capybara'
   gem 'selenium-webdriver'
   gem 'simplecov'
-  gem 'webdrivers'
+  gem 'webdrivers', '~> 5.3', '>= 5.3.1'
```

Comment:
Why the fixed version?

Reply:
Otherwise an error occurs due to a compatibility error between chrome and webdriver